plugin/core/views.py

```diff
@@ -310,7 +310,7 @@ def get_storage_path() -> str:
     Its path is '$DATA/Package Storage', where $DATA means:
 
     - on macOS: ~/Library/Application Support/Sublime Text
-    - on Windows: %AppData%/Sublime Text/Roaming
+    - on Windows: %LocalAppData%/Sublime Text
     - on Linux: $XDG_CONFIG_DIR/sublime-text
```

Comment:
```suggestion
    - on Linux: ~/.cache/sublime-text
```

Reply:
When googling for `XDG_CONFIG_DIR` it doesn't seem to exist. There is `XDG_CONFIG_DIRS` and there is XDG_CONFIG_HOME.

Is `XDG_CONFIG_HOME` defined in your distro? I would imagine that `XDG_CONFIG_HOME` is some standard thing that would work in more cases.

Reply:
It is not defined on my distro (Ubuntu 22.10)
```
λ  ~  echo $XDG_CONFIG_DIR

λ  ~  echo $XDG_CONFIG_HOME

```

Reply:
Actually in my virtual Debian the `XDG_CONFIG_DIRS` seems to be defined but points to `/etc/xdg`.
The `XDG_CACHE_HOME` or `XDG_CONFIG_HOME` are not defined...

Let's just use your suggestion :)